app/src/main/java/org/mozilla/focus/utils/StatusBarUtils.kt

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils
+
+import android.view.View
+
+class StatusBarUtils {
+
+    companion object {
+        private var STATUS_BAR_HEIGHT = -1
+
+        fun getStatusBarHeight(view: View?, listener: (Int) -> Unit) {
+            if (StatusBarUtils.STATUS_BAR_HEIGHT > 0) {
+                listener(STATUS_BAR_HEIGHT)
+            }
+
+            view?.let {
```

Comment:
This is super close. This would work as written, but it'd be cleaner Kotlin code if you simplified a bit. view?.let{} surrounding view.setOnApplyWindowInsetsListener should be identical to just view?.setOnApplyWindowInsetsListener with the safe call operator. https://kotlinlang.org/docs/reference/null-safety.html#safe-calls

Reply:
Oh looking at it again, I agree completely. That is simpler/cleaner. I made the change, tested it on my device, and pushed it up. Thanks!